neuralmonkey/config/builder.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, string):
 
     def create(self):
         class_parts = self.clazz.split(".")
+
```

Comment:
Proč ne, ale vůbec to nepatří do tohodle PR, kterej spravuje typy a docstringy v datasetu.